From 05650624fe2524ecb1bdbbd4facc5d988275e542 Mon Sep 17 00:00:00 2001
From: Dan Callaghan <dcallagh@redhat.com>
Date: Wed, 7 Jun 2017 13:55:53 +1000
Subject: [PATCH] always obey GEVENT_NO_CFFI_BUILD

If libev is not embedded, we still want to set GEVENT_NO_CFFI_BUILD=1 to
prevent setup.py from trying to configure libev, regardless whether cffi
modules will actually be built or not.
---
 setup.py | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/setup.py b/setup.py
index 9c229ad..ea4dcaf 100755
--- a/setup.py
+++ b/setup.py
@@ -90,16 +90,17 @@
 except ImportError:
     pass
 else:
-    if IGNORE_CFFI and not PYPY:
-        # Allow distributors to turn off CFFI builds
-        # even if it's available, because CFFI always embeds
-        # our copy of libev and they may not want that.
-        del cffi_modules[:]
     # Note that we don't add cffi to install_requires, it's
     # optional. We tend to build and distribute wheels with the CFFI
     # modules built and they can be imported if CFFI is installed.
     # install_requires.append('cffi >= 1.3.0')
+    pass
 
+if IGNORE_CFFI and not PYPY:
+    # Allow distributors to turn off CFFI builds
+    # even if it's available, because CFFI always embeds
+    # our copy of libev and they may not want that.
+    del cffi_modules[:]
 
 # If we are running info / help commands, or we're being imported by
 # tools like pyroma, we don't need to build anything
-- 
2.9.4

